### ISSUE TITLE
Cannot update news: TypeError

### ISSUE THREAD

**Issue by tt-tom17** [ioBroker/ioBroker.admin#2166]
**Describe the bug**  
```
Cannot update news: TypeError: newNews.forEach is not a function
```
Die Meldung ist unabhängig von der Version. Auf dem Prod-System läuft 6.10.1 und auf dem Testsystem die aktuelle beta 6.11.0

**To Reproduce**  
Restart Adapter

**Screenshots & Logfiles**  
2023-10-20 14:13:39.524 - info: admin.0 (1265664) starting. Version 6.10.1 in /opt/iobroker/node_modules/iobroker.admin, node: v18.18.2, js-controller: 5.0.12
2023-10-20 14:13:39.606 - info: admin.0 (1265664) requesting all objects
2023-10-20 14:13:42.158 - info: admin.0 (1265664) received all objects
2023-10-20 14:13:42.639 - info: admin.0 (1265664) Request actual repository...
2023-10-20 14:13:42.831 - info: admin.0 (1265664) Secure socket.io server listening on port 8081
2023-10-20 14:13:42.839 - info: admin.0 (1265664) https server listening on port 8081
2023-10-20 14:13:42.840 - info: admin.0 (1265664) Use link "https://127.0.0.1:8081" to configure.
2023-10-20 14:13:42.985 - error: admin.0 (1265664) Cannot update news: TypeError: newNews.forEach is not a function

**Versions:**  
 - Adapter version: stable 6.10.1 / beta 6.11.0
 - JS-Controller version: 5.0.12
 - Node version: 18.18.2
 - Operating system: Raspi bullseye

**Comment by mcm1957** [comment]
duplicate ? 
see https://github.com/ioBroker/ioBroker.admin/issues/2163

**Comment by tt-tom17** [comment]
Ja, schein dasselbe zu sein. Der Fix wird morgen erst im Beta sein?

**Comment by Feuer-sturm** [comment]
@tt-tom17 der fix ist im nächsten Release enthalten. Wann das nächste Release kommt ist abhängig davon, wann es die Entwickler bauen.
In dieser Übersicht siehst du, wenn eine neue Version erzeugt wurde (dort steht dann chore Release...): https://github.com/ioBroker/ioBroker.admin/commits/master
